index.html

```diff
@@ -1,389 +1,462 @@
-                        >
-                            Creatives
-                        </a>
-                    </li>
-                </ul>
-            </nav>
-        </header>
-        <section class="container mx-auto">
-            <section class="py-5 font-work-sans">
-                <h2 class="font-work-sans font-bold pl-8 text-3xl text-center">
-                    About JSLovers
-                </h2>
-                <div
-                    class="flex flex-col justify-center items-center md:px-20 px-5"
-                >
-                    <div class="mb-4">
-                        <p class="text-justify leading-8">
-                            Welcome to <b>JSLovers</b>, the
-                            <a href="#">award-winning</a> JavaScript meetup
-                            group with chapters in <a href="#">Delhi</a>,<a
-                                href="#"
-                                >Bangalore</a
-                            >, and <a href="#">Pune</a>! We are a passionate
-                            community of JavaScript enthusiasts who come
-                            together to share knowledge, explore the latest
-                            trends, and collaborate on exciting projects.
-                        </p>
-                        <p class="text-justify leading-8 mt-4">
-                            Our meetups cover a wide range of topics, from
-                            JavaScript fundamentals to advanced frameworks,
-                            front-end and back-end development, and everything
-                            in between. We believe in the power of networking
-                            and learning from fellow enthusiasts, making our
-                            meetups a great place to connect with like-minded
-                            individuals. JSLovers is proud to have received
-                            recognition from Meetup for our outstanding
-                            contributions to the tech community.
-                        </p>
-                        <p class="text-justify leading-8 mt-4">
-                            We're also grateful to our sponsors, including
-                            Google, Microsoft, and many more, who help make our
-                            events even more exciting and informative. Join us
-                            for our upcoming events, workshops, and discussions
-                            to expand your JavaScript skills and be part of a
-                            supportive community that shares your love for
-                            coding. Let's code, learn, and grow together with
-                            JSLovers!"
-                        </p>
-                        <h2 class="text-3xl py-4 mt-4 font-bold text-center">
-                            Watch Talks About JSLovers
-                        </h2>
+  <body>
+    <a id="top"></a>
```

Comment:
We need to hide it from the flow of the keyboard. As of now, it is taking the first keyboard focus to this tag